dcs-packaging-tool/dcs-packaging-tool-impl/src/test/java/org/dataconservancy/packaging/tool/impl/GeneralPackageDescriptionCreatorTest.java

```diff
@@ -370,6 +371,7 @@ public void simLinkCycleTest() throws Exception {
             Files.createSymbolicLink(link, subdir.toPath());
         } catch (UnsupportedOperationException e) {
             /* Nothing we can do if the system doesn't support symlinks */
+            FileUtils.deleteDirectory(tempDir);
```

Comment:
That's what I thought as well, however, they seem to be only deleted onExit i.e. as long as the JVM and tests are running, they remain and the 4.12 version of TemporaryFolder throws an IOException if a directory exists and fails the test. Did you have any other idea on getting around this?